docker/obdemo-rcs-api/Dockerfile

```diff
@@ -1,11 +1,15 @@
-FROM openjdk:8-jre-alpine
+FROM google/cloud-sdk:alpine
```

Comment:
Change the image `openjdk:8-jre-alpine` to `google/cloud-sdk:alpine` to use `gsutil` instead of `ADD` to avoid get corrupted files from `google bucket`, more fast and secure.